src/Data/Text/Encoding.hs

```diff
@@ -474,33 +474,18 @@ encodeUtf8BuilderEscaped be =
           -- is smaller than 8, as this will save on divisions.
           | otherwise        = return $ B.bufferFull bound op0 (outerLoop i0)
           where
-            outRemaining = (ope `minusPtr` op0) `div` bound
+            outRemaining = (ope `minusPtr` op0) `quot` bound
```

Comment:
This doesn't change anything, does it?

Reply:
Both operands are positive, so the result is unchanged. `quot` usually optimizes better, because it matches CPU instruction for signed division, while `div` requires branches. This is a micro-optimization indeed.